site/app/views/submission/HomeworkView.php

```diff
@@ -197,6 +197,20 @@ public function renderLateDayMessage(LateDays $late_days, Gradeable $gradeable,
             $active_days_charged = max(0, $active_days_late - $extensions);
         }
 
+
+        $daylight_message_required = false;
+        $date = new \DateTime();
+        $daylight_spring = new \DateTime('2024-03-10', $this->core->getConfig()->getTimezone());
```

Comment:
daylight spring and fall should not be hard coded to some date. Could you find a way to update using some dependancies?